analysis2_manuscript.tex

```diff
@@ -914,7 +914,7 @@ \section*{Methods}
     at the time of writing, the library includes over 11,000 lines of testing code,
     that run 2,862 unit tests, including comparisons to QC implementations
     and cross-checks for agreement, including between \slim and \msprime implementations of the same models.
-    For instance, for testing we make use of \slim's scriptability to separately record in simulation metadata
+    For instance, for testing we make use of \slim's scriptability to separately record in-simulation metadata
```

Comment:
Whoops - I think this should probably be removed, as I meant "in metadata of the simulation" and "in-simulation metadata information" is a bit confusing? nbd tho.

Reply:
i can make this change in the PR I'm working on. thanks for the heads up